ksql-rest-app/src/test/java/io/confluent/ksql/rest/entity/SourceDescriptionTest.java

```diff
@@ -90,7 +91,7 @@ public void shouldReturnStatsBasedOnKafkaTopic() {
     StreamsErrorCollector.recordError(APP_ID, kafkaTopicName);
```

Comment:
I think it would make more sense to leave this test be and add a separate test that validates that the metrics are correct. This way the old behavior is still covered, and we can do a more strict check of the metric instances that are returned.

Reply:
ah yes! I had changed this test before I added back the old functionality and forget to add a new test :)